travis/start-database.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
This is a bit annoying - we can't re-use the existing `startDatabase` gradle task/existing scripts because they use the local registry which Travis can't access. 

We can't just switch over to using the public docker hub everywhere because that would mean we could no longer start the database with arbitrary branch or commit tag as per the `db_version` config file (because we only push master and semantic version tags to docker hub.) It feels ok to always use master here because we're just trying to get at code coverage, and tests will be run against the configured version on TeamCity.

So options:
1. maintain 2 ways of starting the database, this script which runs the public master image, and the gradle startDatabase task which runs the configured version image from the local registry
2. push all images to docker hub and stop using our local registry, change the existing task to use the public docker hub and use that everywhere

Reply:
Agreed. Maybe put a comment in here, or rename the file somehow to remind ourselves of why we have two mechanisms (I will definitely forget).

Reply:
Cool, yeah I agree that maintaining the 2 files is probably the better option :+1: